CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 
 ## 1.11.1 (2023-12-07)
 
+### New Features
+
+- Added moving_agg function in DataFrame.transform for time series analysis, enabling moving aggregations like sums and averages with multiple window sizes.
+
```

Comment:
You might have to move the changelog to the latest release candidate

Reply:
done